static/app/routes.tsx

```diff
@@ -242,7 +242,7 @@ function buildRoutes() {
           <IndexRedirect to="welcome/" />
           <Route
             path=":step/"
-            component={make(() => import('sentry/views/onboarding/onboarding'))}
```

Comment:
minor improvement. Please let me know if this change if fine when reviewing this PR